src/include/btree_interval_set.h

```diff
@@ -239,7 +239,7 @@ class btree_interval_set {
     return _size == other._size && m == other.m;
   }
 
-  int size() const {
+  int64_t size() const {
```

Comment:
Because intervet_set is very basic class, some one may hit this bug. Maybe there is no callers in osd which could be impacted by this issue. I am not familiar with mds.
I grep the source code of mds. There are some callers in mds, such as `inos.size()` .